app/models/vm_or_template/scanning.rb

```diff
@@ -8,6 +8,12 @@
 module VmOrTemplate::Scanning
   extend ActiveSupport::Concern
 
+  # Smartstate Analysis is unsupported by default.
+  # Subclasses need to override this method if they support SSA.
+  included do
+    supports_not :smartstate_analysis, :reason => _("Smartstate Analysis is not available for VM or Template.")
```

Comment:
@gauravaradhye Normally, it wouldn't matter that much to me. But we receive numerous complaints about lack of information describing why SSA isn't available in various cases. We've added information over time, so it concerns me when we remove information.

If this message is only displayed when we hover over the disabled button on the detail screen of the VM, maybe we should just change the message to read:
```
"Smartstate Analysis is not available for this type of VM or Template."
```
I think that might be good enough.